benchmarks/dispatch.py

```diff
@@ -0,0 +1,632 @@
+import itertools
```

Comment:
Add a comment to this file to make sure people understand it is for benchmarking locust itself

Reply:
Done in the other PR